src/main/commandServer/httpCommandServer.ts

```diff
@@ -209,23 +185,12 @@ export class HttpCommandServer {
     const chunks: Buffer[] = [];
     let values: Record<string, any> = {};
     let result = '';
+    const [data, payloadError] = await serverHelper.getRequestBody(request, MAX_REQUEST_BODY_LENGTH);
```

Comment:
It looks like we only check that playloadError is falsy, but never write out the actual text?

Reply:
Yes, I might as well shift `payloadError` to `error`, skip calling `updateSettings`, and do the error-reporting in one place.

Reply:
Yes, it's been done at lines 202 and 209.